cesium/db.go

```diff
@@ -122,5 +122,10 @@ func (db *DB) Close() error {
 	for _, u := range db.unaryDBs {
 		c.Exec(u.Close)
 	}
-	return c.Error()
+	if c.Error() != nil {
+		db.closed.Store(true)
```

Comment:
Outdated diff – resolved.

Reply:
For the cesium DB, closed is always swapped to `true` when `Close` is called – this is because even if one unaryDB failed because of open handles, we should still stop serving read/write requests since one of the channels in index/data pairs may be closed while the other is not.